components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.throttling.synchronizer/src/main/java/org/wso2/carbon/apimgt/micro/gateway/throttling/synchronizer/dto/ApplicationThrottlePolicyListDTO.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2017 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.apimgt.micro.gateway.throttling.synchronizer.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@ApiModel(description = "")
+public class ApplicationThrottlePolicyListDTO  {
+  
+  
+  
+  private Integer count = null;
+  
+  
+  private List<ApplicationThrottlePolicyDTO> list = new ArrayList<ApplicationThrottlePolicyDTO>();
+
+  
+  /**
+   * Number of Application throttle policies returned.\n
+   **/
+  @ApiModelProperty(value = "Number of Application throttle policies returned.\n")
+  @JsonProperty("count")
+  public Integer getCount() {
+    return count;
+  }
+  public void setCount(Integer count) {
+    this.count = count;
+  }
+
+  
+  /**
+   **/
+  @ApiModelProperty(value = "")
+  @JsonProperty("list")
+  public List<ApplicationThrottlePolicyDTO> getList() {
+    return list;
+  }
+  public void setList(List<ApplicationThrottlePolicyDTO> list) {
+    this.list = list;
+  }
+
+  
```

Comment:
Let's remove the additional lines here as well. Please do the same for the rest of the classes.